components/brave_wallet/browser/polkadot/polkadot_substrate_rpc.cc

```diff
@@ -208,6 +208,26 @@ base::expected<RpcResponse, std::string> HandleRpcCall(
   return base::ok(std::move(*res));
 }
 
+std::optional<PolkadotBlockHeader> ParseChainHeaderFromHex(
+    const polkadot_substrate_rpc_responses::PolkadotChainHeader& res) {
+  PolkadotBlockHeader header;
```

Comment:
As per the style guide:

> C++ allows you to declare variables anywhere in a function. We encourage you to declare them in a scope as local as possible, and as close to the first use as possible. This makes it easier for the reader to find the declaration and see what type the variable is and what it was initialized to. In particular, initialization should be used instead of declaration and assignment

https://google.github.io/styleguide/cppguide.html#Local_Variables